### PR TITLE
Add a few more view utils

### PR DESCRIPTION
**Pull request by f2rf2r**

### CODE REVIEW

**Comment by prfarlow1** [comment]
:shipit:

File: commonutils/src/main/java/io/intrepid/commonutils/IntrepidViewUtils.java [hunk 1]
```diff
@@ -68,6 +144,106 @@ public static void setPaddingBottom(@NonNull View view, int newPadding) {
...
+    }
+
+    /**
+     * Sets all four sides of a view's margin
+     *
+     * @param view      the input view
+     * @param newMargin the margin in pixel that will be applied to all sides
+     * @throws IllegalArgumentException If the view's parent container does not support margin params
+     */
+    public static void setMargin(@NonNull View view, int newMargin) throws IllegalArgumentException {
```
Comment (weathersboyle) [inline commonutils/src/main/java/io/intrepid/commonutils/IntrepidViewUtils.java]: would it make more sense to call this `setMargins()`?

File: commonutils/src/main/java/io/intrepid/commonutils/IntrepidViewUtils.java [hunk 2]
```diff
@@ -68,6 +144,106 @@ public static void setPaddingBottom(@NonNull View view, int newPadding) {
...
+     * Sets all four sides of a view's margin
+     *
+     * @param view         the input view
+     * @param leftMargin   the new left margin in pixels
+     * @param topMargin    the new top margin in pixels
+     * @param rightMargin  the new right margin in pixels
+     * @param bottomMargin the new bottom margin in pixels
+     * @throws IllegalArgumentException If the view's parent container does not support margin params
+     */
+    public static void setMargin(@NonNull View view,
```
Comment (weathersboyle) [inline commonutils/src/main/java/io/intrepid/commonutils/IntrepidViewUtils.java]: `setMargins()`?

File: commonutils/src/test/java/io/intrepid/commonutils/IntrepidViewUtilsTest.java [hunk 3]
```diff
@@ -25,6 +37,28 @@
 public class IntrepidViewUtilsTest {
 
     @Test
+    public void testSetVisibilities() throws Exception {
+        final View view1 = new View(RuntimeEnvironment.application);
+        final View view2 = new View(RuntimeEnvironment.application);
+        final View view3 = new View(RuntimeEnvironment.application);
+
+        setVisibilities(View.VISIBLE, view1, view2, view3);
```
Comment (weathersboyle) [inline commonutils/src/test/java/io/intrepid/commonutils/IntrepidViewUtilsTest.java:48]: could have utility method, which asserts that given views have given/expected visibility

**Review by weathersboyle — APPROVED** [review]

**Pull request merged**